components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint.util/src/main/java/org/wso2/carbon/identity/mgt/endpoint/util/client/api/RecoveryApiV2.java

```diff
@@ -80,6 +83,35 @@ public void setApiClient(ApiClient apiClient) {
         this.apiClient = apiClient;
     }
 
+    /**
+     * Initiate recovering the forgotten username.
+     *
+     * @param recoveryInitRequest Username recovery initiating request. (required)
```

Comment:
```suggestion
     * @param recoveryInitRequest Username recovery initiating request (required).
```
Generally we add fullstop at end of comment sentence.

Reply:
Fixed by https://github.com/wso2/carbon-identity-framework/pull/6663